src/input/mpegts/linuxdvb/linuxdvb_satconf.c

```diff
@@ -251,14 +251,14 @@ const idclass_t linuxdvb_satconf_class =
     {
       .type     = PT_BOOL,
       .id       = "switch_rotor",
-      .name     = N_("Switch Then Rotor"),
+      .name     = N_("Switch then rotor"),
```

Comment:
As in "Switch is located before rotor"? Or is this a help text thing again to expand upon it?